stack/app/README.md

```diff
@@ -82,7 +82,7 @@ module "stack" {
       bucket_name                     = "project-staging-media",
       versioning                      = true,
       primary_storage_class_retention = 0,
-      replicas                        = []
+      # replicas is an optional variables, it can be omitted
```

Comment:
```suggestion

      # optional      
      replicas = []
```

Reply:
to be consistent with other readmes. Also easier to read and shows the type (array).